File: src/main/java/net/im45/bot/chain/Chain.java
package net.im45.bot.chain;

import net.mamoe.mirai.console.command.BlockingCommand;
import net.mamoe.mirai.console.command.CommandSender;
import net.mamoe.mirai.console.command.JCommandManager;
import net.mamoe.mirai.console.plugins.PluginBase;
import org.jetbrains.annotations.NotNull;

import java.util.List;

public class Chain extends PluginBase {
    @Override
    public void onLoad() {
        JCommandManager.getInstance().register(this, new BlockingCommand(
                "chain",
                List.of(),
                "Chain!",
                "chain"
        ) {
            @Override
            public boolean onCommandBlocking(@NotNull CommandSender commandSender, @NotNull List<String> list) {
                return false;
            }
        });
    }
}

File: src/main/kotlin/net/im45/bot/chain/Chain.kt
package net.im45.bot.chain

import com.google.auto.service.AutoService
import net.mamoe.mirai.console.plugin.jvm.JvmPlugin
import net.mamoe.mirai.console.plugin.jvm.JvmPluginDescription
import net.mamoe.mirai.console.plugin.jvm.KotlinPlugin

@AutoService(JvmPlugin::class)
object Chain : KotlinPlugin(
    JvmPluginDescription(
        "net.im45.bot.chain",
        "0.1.0"
    )
) {
    override fun onEnable() {
        super.onEnable()
    }
}
